homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,12 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    if(count <= 0 ) {
+        return {};
+    }
+    std::vector<int> vec{step};
```

Comment:
Reserve memory when you know how many elements will be inside vector.

Reply:
Here you create vec with one element, this is misleading because in loop below you add another elemnts.